rest_framework/parsers.py

```diff
@@ -22,6 +22,7 @@
 
 from rest_framework import renderers
 from rest_framework.exceptions import ParseError
+import codecs
```

Comment:
PEP8: Import should be grouped in order: standard lib, 3rd party, local
https://www.python.org/dev/peps/pep-0008/#imports